workflow/rules/common.smk

```diff
@@ -400,9 +400,7 @@ def get_contigs(wildcards):
 def get_expanded_contigs(wildcards):
     sample = get_samples_for_date(wildcards.date)
     return [
-        "results/{{date}}/assembly/{sample}/{assembler}/{sample}.contigs.fasta".format(
-            sample=s, assembler=return_assembler(s)
-        )
+        "results/{{date}}/contigs/checked/{sample}.fasta".format(sample=s)
```

Comment:
Nice change here! But apparently the function `get_expanded_contigs` is just used in the rule `virologist_report`. Thus, `get_expanded_contigs` is kind of obsolete now. Please remove it completely and just use `expand_samples_for_date` instead.